docker-compose.yml

```diff
@@ -15,8 +15,6 @@ services:
       - app
     ports:
       - '3306'
-    volumes:
-      - .docker/drupal/dumps:/docker-entrypoint-initdb.d
```

Comment:
Has this been removed on purpose? I'm not sure that we use it or that the database dump is up-to-date, so just removing it may be the right thing to do.

Reply:
A wise man showed me how to sync the docker env. with itkdev-docker-compose self:update .

https://github.com/itk-dev/devops_itkdev-docker/blob/develop/templates/drupal-10/docker-compose.yml